helpers/DeployHelper.ts

```diff
@@ -563,7 +565,7 @@ export class DeployHelper {
     return g16Verifier;
   }
 
-  getGroth16VerifierWrapperName(verifierType: "mtpV2" | "sigV2" | "v3"): string {
+  getGroth16VerifierWrapperName(verifierType: "mtpV2" | "sigV2" | "v3" | "authV2"): string {
```

Comment:
The Typescript type "mtpV2" | "sigV2" | "v3" | "authV2" should be defined in one place  and reused

Reply:
fixed